contracts/eden/src/migrations.cpp

```diff
@@ -43,6 +42,7 @@ namespace eden
              state);
       }
       migration_sing.set(state, contract);
+      push_event(migration_event{static_cast<varuint32>(state.index())}, contract);
```

Comment:
This can cause the contract to send multiple events with the same index.  Is that what we want?

Reply:
Nice catch, it's not, I'm fixing it to be triggered only when the event actually finishes.